tests/test_user_views.py

```diff
@@ -0,0 +1,44 @@
+from unittest import TestCase
+from models import db, User, Favorites, Match
+from run import app
+
+app.config["WTF_CSRF_ENABLED"] = False
```

Comment:
@iTaylorCodes move this to the config file.

Reply:
Will do 👍